docs/operate/get-started/other-hardware/_index.md

```diff
@@ -843,13 +845,7 @@ viam module reload <insert relevant named args>
 {{% /tab %}}
 {{% tab name="Python: venv" %}}
 
-Run the following command to rebuild your module:
-
-```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
-viam module build local
-```
-
-Then restart it in your machine's **CONFIGURE** tab in the Viam app.
+Restart the module in your machine's **CONFIGURE** tab in the Viam app.
```

Comment:
so for pyinstaller I need to run the CLI command but for venv I need to do it in the UI? What happens if I use the restart option in the UI for pyinstaller and the cli command for venv?

Reply:
So they're not actually the same--reload rebuilds AND restarts, whereas the UI just restarts. So, reload is unnecessarily time consuming if using run.sh as the entrypoint, because nothing needs to be built.

If you use the restart button for pyinstaller, you won't have a newly-built executable--you'll just be restarting the same executable you used previously.

Reply:
Do you think I should make this more explicit? Didn't want to over-explain and make it to verbose, but rather just tell them what they need to do.

Reply:
Yeah I think it may just be a bit of a tripwire where people see `viam module build local` on the first tab and expect to be able to do the same on venv. I think a sentence about "Because you're using pyinstaller it is enough to rebuild the module with: ..." and soemthing similar on venv would avoid any stumbling

Reply:
Okay added something to the venv tab but I think the first tab is okay as-is now that the venv tab addresses why it's different.